### ISSUE TITLE
--url is required when enrolling a fleet server with a cert

### ISSUE THREAD

**Issue by ppf2** [elastic/beats#27245]
In https://www.elastic.co/guide/en/fleet/7.14/elastic-agent-cmd-options.html#_examples, if you try the `Start Elastic Agent with Fleet Server (running on a custom CA)` example:

```
elastic-agent enroll -f --fleet-server-es=https://elasticsearch:9200 \
  --fleet-server-es-ca=ca.crt --fleet-server-service-token=AbEAAdesYXN1abMvZmxlZXQtc2VldmVyL3Rva2VuLTE2MTkxMzg3MzIzMTg7dzEta0JDTmZUcGlDTjlwRmNVTjNVQQ \
  --fleet-server-cert fleet-server.crt --fleet-server-cert-key fleet-server.key
```

It will fail and tell you that URL is required because a certificate is specified.  We should include `--url` in the example commands that contain a cert.

**Comment by dedemorton** [comment]
Noting here that https://github.com/elastic/observability-docs/pull/1091 fixed this problem for 7.15 and later, but I forgot to backport it to 7.14.  I'll leave this issue open until it's fully backported.

**Comment by dedemorton** [comment]
Closed by elastic/observability-docs#1263
